.github/workflows/dockerhub.yml

```diff
@@ -26,16 +30,20 @@ jobs:
     runs-on: ubuntu-latest
     env:
       name: ${{ inputs.Application }}
+    outputs:
+      docker_tag: ${{ steps.build.outputs.docker_tag }}
```

Comment:
This adds the `docker_tag` output to the _job_ so that we can reference it in the workflow outputs above (`${{ jobs.DockerHub.outputs.docker_tag }}`)